sparse_strips/vello_bench/src/cpu_fine.rs

```diff
@@ -96,7 +96,7 @@ pub fn fill(c: &mut Criterion) {
                 },
                 stops: $stops,
                 extend: peniko::Extend::$extend,
-                transform: Affine::default(),
+                ..Default::default()
```

Comment:
There's a periodic lint that we do that will ask us to turn this into `Gradient::default()`, so might as well do it now. Below as well (and in other files).

Reply:
The proper style for `..Default::default()` is this one, as far as I know. That is, the editor periodic lint doesn't trigger in this case.